gulp/tasks/scripts.js

```diff
@@ -84,13 +84,31 @@ function scriptsOnDemand() {
     } ) );
 }
 
+function scriptsEs5Shim() {
```

Comment:
Minor: Add jsdoc above. Return type is `@returns {PassThrough} A source stream.` I'm pretty sure.